lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -218,7 +222,8 @@ export default function getRecipeInterface(
                 response.session.userId,
                 payload,
                 undefined,
-                response.accessToken !== undefined
+                response.accessToken !== undefined,
+                payload.tId
```

Comment:
we need to get tId from the response object which comes from `SessionFunctions.getSession`. `SessionFunctions.getSession` in turn calls `getInfoFromAccessToken` which should extract the tId if it's version >= 4, else set it to `public`

Reply:
Done